### DISCUSSION TITLE
debian-base:bullseye-v1.2.0 this image where i can download?

### DISCUSSION THREAD

**Discussion by linpanru** [kubernetes-csi/csi-driver-nfs#436] [Q&A] [answered]
debian-base:bullseye-v1.2.0 this image where i can download ?

**Comment by andyzhangx** [comment, marked as answer]
docker pull registry.k8s.io/build-image/debian-base:bullseye-v1.2.0

**Reply by linpanru** [reply]
> docker pull registry.k8s.io/build-image/debian-base:bullseye-v1.2.0

i can not visit registry.k8s.io this registry, are there any other registries available?

**Reply by andyzhangx** [reply]
push this image to dockerhub?

**Reply by linpanru** [reply]
> push this image to dockerhub?

brother, now, i have not this image ，can you push this image to dockerhub ,than i download, thanks !

**Reply by andyzhangx** [reply]
try this `andyzhangx/debian-base:bullseye-v1.2.0`

**Reply by linpanru** [reply]
> try this `andyzhangx/debian-base:bullseye-v1.2.0`

it is ok, thanks very much , and i also need  linux/arm64, do you have ? my dear brother

**Reply by andyzhangx** [reply]
sorry I don't have. I think you need to set up VPN to pull that image

**Reply by linpanru** [reply]
> sorry I don't have. I think you need to set up VPN to pull that image

ok，thanks very much and i will try VPN
